cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/service/http/BodyConsumerAdapter.java

```diff
@@ -173,6 +177,7 @@ public void run(DatasetContext context) throws Exception {
       }
     } catch (Throwable t) {
       responder.setTransactionFailureResponse(t);
+      LOG.debug("Exception in calling HttpContentConsumer.onError", t);
```

Comment:
Why not set this to WARN? I think majority of the users would want to see these logs by default, since they shouldn't be happening regularly and only in unexpected scenarios.